share/shwap/p2p/shrex/shrexnd/params.go

```diff
@@ -5,22 +5,22 @@ import (
 
 	logging "github.com/ipfs/go-log/v2"
 
-	"github.com/celestiaorg/celestia-node/share/p2p"
+	"github.com/celestiaorg/celestia-node/share/shwap/p2p/shrex"
 )
 
-const protocolString = "/shrex/nd/v0.0.3"
+const protocolString = "/shrex/nd/v0.0.4"
```

Comment:
feels like a minor bump to me

Reply:
Just curious: do we've a policy why and how should we bump such protocol string?

Reply:
We adhere to semver. This change is protocol-breaking, so it should be minor(as we v0 still) or can be a major bump, that I am not opposed to.